### PR TITLE
Update docs for cri-tools

### PR DESCRIPTION
**Pull request by feiskyer**
Documentation for https://github.com/kubernetes/features/issues/292.

### CODE REVIEW

**Comment by idvoretskyi** [comment]
cc @kubernetes/sig-node-pr-reviews

File: contributors/devel/cri-validation.md [hunk 1]
```diff
@@ -34,15 +41,13 @@ This will
 - Run the tests using `ginkgo`
 - Output the test results to STDOUT
 
-critest connects to `/var/run/dockershim.sock` by default. For other runtimes, the endpoint can be set in two ways:
-
-- By setting flags `--runtime-endpoint` and `--image-endpoint`
-- By setting environment variables `CRI_RUNTIME_ENDPOINT` and `CRI_IMAGE_ENDPOINT`
+critest connects to `unit:///var/run/dockershim.sock` by default. For other runtimes, the endpoint can be set by flags `-runtime-endpoint` and `-image-endpoint`.
```
Comment (yujuhong) [inline contributors/devel/cri-validation.md]: s/unit/unix

Is it `-runtime-endpoint` or `--runtime-endpoint`?

File: contributors/devel/cri-validation.md [hunk 2]
```diff
@@ -34,15 +41,13 @@ This will
 - Run the tests using `ginkgo`
 - Output the test results to STDOUT
 
-critest connects to `/var/run/dockershim.sock` by default. For other runtimes, the endpoint can be set in two ways:
-
-- By setting flags `--runtime-endpoint` and `--image-endpoint`
-- By setting environment variables `CRI_RUNTIME_ENDPOINT` and `CRI_IMAGE_ENDPOINT`
```
Comment (yujuhong) [inline contributors/devel/cri-validation.md]: Were the environment variables removed, and why?

Comment (feiskyer) [reply contributors/devel/cri-validation.md]: Yep, those environment variables are introduced with lib `github.com/urfave/cli`. But with this lib, critest requires Go installed whenever running validation tests. So we moved to `go test -c` for building critest binary and it doesn't require Go for running critest.

File: contributors/devel/cri-validation.md (hunk 1 above)
Comment (feiskyer) [reply contributors/devel/cri-validation.md]: ACK, those options are Go flags, so `-runtime-endpoint` is right. Also additional flags are in same format.

**Comment by feiskyer** [comment]
@yujuhong Addressed comments. PTAL

**Comment by feiskyer** [comment]
@cblecker Could you help to approve the PR?

**Pull request merged**